docs/buildah-push.md

```diff
@@ -40,6 +40,10 @@ Image stored in local container/storage
 
 ## OPTIONS
 
+**--authfile** *path*
+
+Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json
```

Comment:
IDK, since this file is created by "Buildah" should we mention that the file is created via kpod-login?

Reply:
fixed